webpack.config.js

```diff
@@ -1,22 +1,22 @@
 const path = require('path');
 
 module.exports = {
-  entry: ['./browser/src/react-redux/index.jsx'],
+  entry: ['./browser/src/react-redux/index.react.js'],
```

Comment:
I'm actually a fan of index files not containing the extension `.react.js`. Let's just change this to `index.js` (and rename the actual index file as well).